projects/ViTDet/REMEAD.md

```diff
@@ -0,0 +1,101 @@
+# ViTDet
+
+## Description
+
+This is an implementation of [ViTDet](https://github.com/facebookresearch/detectron2/tree/main/projects/ViTDet) based on [MMDetection](https://github.com/open-mmlab/mmdetection/tree/3.x), [MMClassification](https://github.com/open-mmlab/mmclassification/tree/1.x), [MMCV](https://github.com/open-mmlab/mmcv), and [MMEngine](https://github.com/open-mmlab/mmengine).
+
+## Usage
+
+### Training commands
+
+In MMDetection's root directory, run the following command to train the model:
+
+```bash
+python tools/train.py projects/ViTDet/configs/vitdet_mask-rcnn_vit-b-mae_lsj-100e.py
+```
+
+### Testing commands
+
+In MMDetection's root directory, run the following command to test the model:
+
+```bash
+python tools/train.py projects/ViTDet/configs/vitdet_mask-rcnn_vit-b-mae_lsj-100e.py ${CHECKPOINT_PATH}
+```
+
+## Results
+
+Based on mmdetection, this project aligns the test accuracy of the [ViTDet](https://github.com/facebookresearch/detectron2/tree/main/projects/ViTDet).
+<br>
+If you want to reproduce the test results, you need to convert model weights first, then run the test command.
```

Comment:
might need update

Reply:
tiny thing, but noticed README is spelled REMEAD 😝

Reply:
sorry, the new version will update